freshbooks/models.py

```diff
@@ -141,6 +142,19 @@ def __str__(self) -> str:
     def __repr__(self) -> str:  # pragma: no cover
         return "ListResult({})".format(self._name)
 
+    def __add__(self, other: "ListResult") -> "ListResult":
```

Comment:
Shouldn't it be just `ListResult` instead of `"ListResult"`?

Reply:
It gets unhappy because `ListResult` isn't a defined class until after this gets run. Looking online the suggestion was to quote it.

Reply:
https://stackoverflow.com/a/36286947